quad_ad5780.py

```diff
@@ -75,9 +75,18 @@ def shutdown(self):
     def do_init(self):
         yield self.packet().write("INITIALIZE\r").send()
         p=self.packet()
-        p.read_line()
         ans = yield p.send()
-        returnValue(ans.read_line)
+
+        done = False # wait for "INITIALIZATION COMPLETE"
+        while not done:
```

Comment:
Have you been having trouble initializing the quad?  `p.send()` is a blocking call; it'll wait until it gets a response or until the quad timeouts. 
`while not done:`  If the quad never initializes , it'll go into an infinite loop.